_docs/2.0.0/usage/menas-api.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: docs
+title: Usage - Menas API
+version: '2.0.0'
+categories:
+    - '2.0.0'
+    - usage
+---
+
+- [Export Entity](#export-entity)
+  - [Example Import Response for Schema](#example-import-response-for-schema)
+- [Entity Import](#entity-import)
+  - [JSON payload](#json-payload)
+  - [Example Export Response for Schema](#example-export-response-for-schema)
+
+
+### Export Entity
+
+This endpoint exports a single entity from Menas. All exports are versionless, except for connected entities. They are kept as is, but need to be validated before import.
+
+`GET {menas_url}/api/{entity}/exportItem/{name}/{version}`
+
+- entity is `schema`, `dataset` or `mappingTable`
+- name is the name of the entity as seen in Menas
+- version is the version of the entity as seen in Menas. Version is optional and in that case the latest version will be used
+
+#### Example Import Response for Schema
+
+Returns code is 200.
```

Comment:
This does not belong to the examples chapter, I bet. But few lines higher, I guess.

Also no 404 status or maybe other?